### PR TITLE
fix: github sponsors

### PR DESCRIPTION
**Pull request by eddiejaoude**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1725"></a>

### CODE REVIEW

File: public/data/eddiejaoude.json [hunk 1]
```diff
@@ -43,6 +43,11 @@
       "name": "Listen to my Podcast",
       "url": "https://anchor.fm/eddiejaoude",
       "icon": "codeforces"
+    },
+    {
+      "name": "Donate to me on GitHub Sponsors",
+      "url": "https://github.com/sponsors/eddiejaoude",
+      "icon": "github"
```
Comment (Pradumnasaraf) [inline public/data/eddiejaoude.json:50]:
```suggestion
      "icon": "dollar"
```

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by Pradumnasaraf — COMMENTED** [review]
Hey, Eddie, LinkFree also has [dollar](https://github.com/EddieHubCommunity/LinkFree/blob/main/icons.md) icon and I think it fits well. What are your thoughts?

File: public/data/eddiejaoude.json (hunk 1 above)
Comment (eddiejaoude) [reply public/data/eddiejaoude.json]: Great, thank you 👍

**Review by Panquesito7 — APPROVED** [review]
Looks great! 🚀


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

File: public/data/eddiejaoude.json [hunk 2]
```diff
@@ -43,6 +43,16 @@
...
       "icon": "codeforces"
+    },
+    {
+      "name": "Donate to me on GitHub Sponsors",
+      "url": "https://github.com/sponsors/eddiejaoude",
+      "icon": "dollar"
+    },
+    {
+      "name": "Customising your GitHub Profile course (free)",
+      "url": "http://eddiejaoude.io/course-github-profile-landing",
```
Comment (kumarsonsoff3) [inline public/data/eddiejaoude.json:54]: It's using the secure extension, so I believe we should update it!! 
```suggestion
      "url": "https://eddiejaoude.io/course-github-profile-landing",
```

Comment (eddiejaoude) [reply public/data/eddiejaoude.json:54]: Thank you, but the link is without https and it redirects to the https on a sub domain

**Pull request merged**